src/styleUtils.ts

```diff
@@ -0,0 +1,102 @@
+import {processColor, Platform} from 'react-native';
+import type * as MarkdownTextInputDecoractorView from './MarkdownTextInputDecoratorViewNativeComponent';
+
+type MarkdownStyle = MarkdownTextInputDecoractorView.MarkdownStyle;
+
+type PartialMarkdownStyle = Partial<{
+  [K in keyof MarkdownStyle]: Partial<MarkdownStyle[K]>;
+}>;
```

Comment:
Let's add a newline here
```suggestion
}>;

```